### ISSUE TITLE
search only works in current window

### ISSUE THREAD

**Issue by leggewie** [Quicksaver/Tab-Groups#275]
Thank you VERY much for the awesome work.

It seems that unlike "previous" versions, the search function only works in the current window now.  Before it would search all tabs in all groups in all windows.

**Comment by Quicksaver** [comment]
Yes, unfortunately as I explained in the release notes, results from other windows had to be cut, they actually weren't 100% reliable. I plan on re-implementing these (properly) when I work on things like #82 and #174.

**Comment by Quicksaver** [comment]
I actually hadn't meant to close this.

**Comment by leggewie** [comment]
@Quicksaver Can you reopen this ticket, then?

**Comment by Quicksaver** [comment]
This is just weird, I had done it when I commented... pretty sure...

**Comment by pholbrook** [comment]
Everyone is a special case. For me, the main reason I use tab groups is that search feature - I know I have the tab open someplace, but can't remember where. I eagerly look forward to the return of this feature.
